op-node/rollup/driver/state.go

```diff
@@ -195,6 +201,11 @@ func (s *Driver) eventLoop() {
 		sequencerTimer.Reset(delay)
 	}
 
+	// Create a ticker to check if there is a gap in the engine queue every minute
+	// If there is, we send requests to the backup RPC to retrieve the missing payloads
+	// and add them to the unsafe queue.
+	altSyncTicker := time.NewTicker(60 * time.Second)
```

Comment:
I think we want a `defer altSyncTicker.Close()` here to prevent a memory leak

Reply:
Re-assigned to 15 seconds.